turbinia/config/__init__.py

```diff
@@ -44,13 +52,18 @@
 # Environment variable to look for path data in
 ENVCONFIGVAR = 'TURBINIA_CONFIG_PATH'
 
+CONFIG = None
 
 class TurbiniaConfigException(Exception):
   pass
 
 
 def LoadConfig():
   """Finds Turbinia config file and loads it."""
+  global CONFIG
```

Comment:
As discussed, possibly with a singleton pattern on the config class.  I put in a TODO for now.